vimeo-networking/src/main/java/com/vimeo/networking/model/Video.java

```diff
@@ -483,12 +490,14 @@ public Connection getRelatedConnection() {
         return null;
     }
 
-    public int likeCount() {
-        if (getLikesConnection() != null) {
-            return getLikesConnection().mTotal;
+    @Nullable
+    public Connection getTexttracksConnection() {
+        if ((mMetadata != null) && (mMetadata.mConnections != null) && (mMetadata.mConnections.mTexttracks != null)) {
+            return mMetadata.mConnections.mTexttracks;
```

Comment:
Could you rename `mTexttracks` to `mTextTracks` as well as the method name, to match the class that we have? I'd like to see consistent naming around `Texttracks` vs `TextTracks`.